geom/src/circle.rs

```diff
@@ -71,16 +71,16 @@ impl Circle {
         )
     }
 
-    pub fn outline(center: Pt2D, radius: Distance, thickness: Distance) -> Result<Polygon, String> {
-        if radius <= thickness {
+    pub fn to_outline(&self, thickness: Distance) -> Result<Polygon, String> {
```

Comment:
From other UI development (web and iOS), I'm used to borders being applied in *addition* to the things internal size - e.g. a 20px square with a 3px border will occupy 3px + 20px + 3px = 26px² square. 

Is it consistent with elsewhere in the application that borders are subtracted so they don't affect the overall size of the thing getting the border?

Reply:
This new `to_outline` "builder" style api is an improvement vs passing everything into the previous constructor style method. :+1: